.github/workflows/ci.yml

```diff
@@ -258,6 +261,7 @@ jobs:
         with:
           path: src/github.com/containerd/containerd
           repository: "containerd/containerd"
+          ref: "release/1.7"
```

Comment:
Should this be like v1.7.0 or something? I imagine the release branch will be updated regularly. Do we want that here specifically?

Reply:
the alternative is that we have to update pretty regularly, which feels like a trade off
i think it should be fine to always pull the latest tests